packages/vite-node/src/cli.ts

```diff
@@ -24,6 +28,7 @@ export interface CliOptions {
   root?: string
   config?: string
   watch?: boolean
+  serverOptions?: ViteNodeServerOptions
```

Comment:
Since the option is provided by the consumer, it might not match `ViteNodeServerOptions`, so I'm cheating a little bit here.

However, we could rely on a TS guard to assert that the given options match the TS interface, but it would mean hardcoding the checks, and thus duplicating the source of truth between `ViteNodeServerOptions` and the TS guard. Concretely, both could be out of sync, leading to issues. 

Do you see another way to achieve this without duplicating the source of truth? 🤔